src/Exception/BasicException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace MatthiasMullie\Minify\Exception;
+
+/**
+ * @author Matthias Mullie <minify@mullie.eu>
+ */
+class BasicException extends \Exception
```

Comment:
Unfortunately, we can't get rid of the original MatthiasMullie\Minify\Exception class. In retrospect, I wish I had properly segmented exceptions initially, but now that's too late.
People could be catching exceptions of class MatthiasMullie\Minify\Exception, so we can't just remove it without breaking backwards compatibility. I'll keep all of these classes around, but will make sure they still inherit from the original base exception class.